bin/convert-argv.js

```diff
@@ -441,7 +441,10 @@ module.exports = function(...args) {
 
 		ifArg("output-library", function(value) {
 			ensureObject(options, "output");
-			options.output.library = value;
+			if (!options.output.library) {
```

Comment:
Default value could be an empty array from `yargs`

Reply:
Just tried this, does not work. Default value is not set if a value is given via the CLI - it isn't an initial value. However I did replace this code w/ `ensureArray`